src/config/default-app-config.ts

```diff
@@ -364,4 +364,8 @@ export class DefaultAppConfig implements AppConfig {
     enableEndUserAgreement: true,
     enablePrivacyStatement: true
   };
+
+  enableMarkdown = false;
+
+  enableMathjax = false;
```

Comment:
I'm wondering if we should move these under `item` above, so..

```
item: ItemConfig = {
   ...
   // Enable Markdown (https://commonmark.org/) syntax in Item metadata fields
   // By default only dc.description.abstract supports Markdown.
   enableMarkdown = false;
   // Enable MathJax (https://www.mathjax.org/) syntax in Item metadata fields
   // Requires `enableMarkdown` to also be enabled before MathJax will display.
   enableMathjax = false;
}
```


We also should add these to the `config.example.yml` and add a description for each (I suggested possible descriptions above in my comments).

Reply:
I'm not sure about moving he configuration to `item`; I can imagine that etc. home news or Comm/Coll descriptions are likely places users would want to have Markdown support going forward

→ not a high priority now, but I wouldn't steer away from the possibility by moving the config

Reply:
@tdonohue @ybnd I think that this could be moved, as Tim suggests, inside `ItemConfig`, but keeping different settings for each metadata.

For example:

```typescript
item: ItemConfig = {
  enableMarkdown: {
    abstract: true,
    description: false,
    other: false,
  }
  enableMathjax: {
    // ...
  }
}
```

Or, alternatively:

```typescript
item: ItemConfig = {
  enableMarkdown: [ 'dc.description'],
  enableMathjax: ['dc.description', 'dc.description.abstract']
}
```

Reply:
I've moved the markdown-related configs to a new interface 'MarkdownConfig':

  markdown: {
    enabled: false,
    mathjax: false,
  };

I've also added some docs to the MarkdownConfig.

I think we need to distinguish between two things:
* the markdown config: controls what the MarkdownPipe does, wherever it is used
  * enabled: if false, do nothing
  * mathjax: render mathjax (only works if enabled above is true)
* the item page metadata: determine whether the MarkdownPipe is used for a specific metadata field
  * this is determined by the 'enableMarkdown' field on the MetadataValuesComponent
  * this field will determine whether the MarkdownPipe is used at all for a metadata field.
  * if markdown has to be enabled for a metadata field, a dedicated component has to be created for this metadata field (if it doesn't exist yet) and enableMarkdown has to be set to 'true' for this component, similar to how it is done for the enableMarkdown
  * adding config options to the 'item' config to control for which metadata fields the markdown pipe is to be used, like @davide-negretti suggested, could also be an option, but at this point we decided not to do that, to keep the templates in control of where the pipe is used
* the MarkdownPipe is currently only used for item metadata, but it could at a later stage also be used at other places, e.g. community/collection metadata, static pages, etc.